Sources/RSDKUtils/Extensions/ASIdentifierManager+Extensions.swift

```diff
@@ -0,0 +1,11 @@
+import AdSupport.ASIdentifierManager
+
+protocol AdvertisementIdentifiable {
+    var advertisingIdentifierUUIDString: String { get }
+}
+
+extension ASIdentifierManager: AdvertisementIdentifiable {
```

Comment:
Is this useful outside of Analytics? I'm not so sure.

Reply:
You are probably right. `SwiftyDependenciesContainerSpec` uses them but maybe we can use something else instead. What do you think @malayli ? I don't want to spoil anyone's work

Reply:
I prefer to move a class/struct/enum to SDKUtils only if it is needed by other SDKs. What do you think?
Is ASIdentifierManager needed by other SDKs? @imaginaris

Reply:
@malayli No, it is not. Ok I'll remove that and fix the tests